src/test/java/http_server/DirectoryCreatorTest.java

```diff
@@ -1,11 +1,25 @@
 package http_server;
 
-import org.junit.Test;
-
+import file_handler.FileHandler;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import java.util.List;
-import static org.junit.Assert.assertEquals;
 
 public class DirectoryCreatorTest {
+    @BeforeEach
```

Comment:
Can you think of a way to test this functionality without creating real files in your file system?